ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -240,138 +240,137 @@
   </Form::Field::ListWrapper>
 
   {{! Account Claim Maps }}
-  <Rose::Form::Fieldset>
-    <:title>{{t 'form.account_claim_maps.label'}}</:title>
-    <:description>{{t 'form.account_claim_maps.help'}}</:description>
-    <:body>
-      {{#if @model.errors.account_claim_maps}}
-        <Rose::Form::Errors as |errors|>
-          {{#each @model.errors.account_claim_maps as |error|}}
-            <errors.message>{{error.message}}</errors.message>
-          {{/each}}
-        </Rose::Form::Errors>
-      {{/if}}
-      <Rose::Table @style='condensed' as |table|>
-        <table.header as |header|>
-          <header.row as |row|>
-            <row.headerCell>{{t 'form.from_claim.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.to_claim.label'}}</row.headerCell>
-            <row.headerCell>{{t 'titles.actions'}}</row.headerCell>
-          </header.row>
-        </table.header>
-        <table.body as |body|>
-          {{#each @model.account_claim_maps as |claimMap i|}}
-            <body.row as |row|>
-              <row.cell>
-                <form.input
+  <Hds::Form::Fieldset @isOptional={{true}} as |F|>
+    <F.Legend>{{t 'form.account_claim_maps.label'}}</F.Legend>
+    <F.HelperText>{{t 'form.account_claim_maps.help'}}</F.HelperText>
+    {{#let (unique-id) (unique-id) as |fromHeaderID toHeaderID|}}
+      <F.Control>
+        <Hds::Table class='list-wrapper-field' name='account_claim_maps'>
+          <:head as |H|>
+            <H.Tr>
+              <H.Th id={{fromHeaderID}}>{{t 'form.from_claim.label'}}</H.Th>
```

Comment:
@calcaide we can remove this, was going to use it in testing and never needed it.

Reply:
I saw this new implementation creates `fromHeaderID` and `toHeaderID` in line 246 and I am wondering what's the reason behind these Id's, are just for testing purposes?

Reply:
@calcaide I have a limited understanding of `(unique-id)` helper, but in this case, because we are not using the standardized way of the table with using `@columns` argument so therefore we loose control over the `a11y` for the inputs and select field labels?